### DISCUSSION TITLE
Installation et exécution de cli-chess

### DISCUSSION THREAD

**Discussion by maulyo** [trevorbayless/cli-chess#36] [Q&A] [answered]
Bonjour,
J'ai installé cli-chess avec cette commande `pip install cli-chess`. Cependant, si je tape dans le terminal `cli-chess` pour le lancer, j'obtient cette erreur :
'cli-chess' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.
Pourriez-vous m'aider ?

**Comment by trevorbayless** [comment]
Hello! What operating system are you trying to install cli-chess on?

**Comment by maulyo** [comment]
Hello, I want to install cli-chess on Windows 10. Thanks you

**Comment by trevorbayless** [comment, marked as answer]
This is likely due to you not having the entry where pip installs scripts added to your windows `Path` environment variable. For example, I tried installing on Windows and received this message after it downloaded:

![image](https://github.com/user-attachments/assets/f9ae89ae-85fa-4e7a-b136-576b3275344f)

You likely received a similar message. To get it working, do the following:

- Copy the python command path from your output (similar to what I posed above) and add it to your `Path` environment variable. In order to see the WARNING message again, you will likely need to `pip uninstall cli-chess` and then `pip install cli-chess` again.
- After editing the `Path` variable, _close the terminal and re-open it (IMPORTANT)_.
- Type `cli-chess` in the command prompt and it should launch normally!

**Reply by maulyo** [reply]
I have succeeded in launching cli-chess ! Thanks you for your help !!
